### DISCUSSION TITLE
Stable Diffusion - Guys make a software, like GIMP

### DISCUSSION THREAD

**Discussion by skudcoding** [comfyanonymous/ComfyUI#2470] [Ideas]
Sir, it will be a first and a giant step from SD community to come out with a new App / software similer like GIMP + KRITA where everyone is able to generate , edit, composite, animate + A.I. with any image or video. I wish you guys will come out with such a tool in near future. I am a regular KRITA user and learning comfyUI. The AI server doesnt connect with krita smoothly. I think krita should have some offline AI features.  Thank you guys....

**Comment by stephantual** [comment]
Ok here's just my humble opinion:
a) no doubt we need a better canvas editor. We do have one of course with https://github.com/Lerc/canvas_tab and it's not bad, but having the option to insert multiple images in a way similar to krea would be fantastic!
b) no doubt masks and very detailled work is best done in krita. Integration is key, i had no issue with https://github.com/Acly/krita-ai-diffusion - is that the one you're using?

My two cents; we're getting there. it's headed there - comfy as the ultimate end-to-end AI suite . For now, we have to sometimes pop out of our workflows to do keying work.
